### ISSUE TITLE
UTXO containing Atomicals cannot be used!

### ISSUE THREAD

**Issue by JD-Wang** [unisat-wallet/extension#243]
How to use UTXO that is occupied by Atom? Atom is already obsolete, why support it? Can you give a switch to turn off restrictions on Atom?

**Comment by cloud6605** [comment]
Yes, we're planning to allow users to use arc20. But it needs time to finish it. Please tell us your wallet address, we'll have a check then give you further help. Or you may open a ticket at our official discord. @JD-Wang

**Comment by GGG888GGG** [comment]
@JD-Wang you could upgrade to the latest version like extension v1.6.0, which can not support arc20 protocol.
